core/src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -926,7 +926,26 @@ class DeltaSuite extends QueryTest
       )
     }
   }
-
+  test("get touched files for update, delete and merge") {
+    withTempDir { dir =>
+      val directory = new File(dir, "test with space")
+      val df = Seq((1, 10), (2, 20), (3, 30), (4, 40)).toDF("key", "value")
+      val writer = df.write.format("delta").mode("append")
+      writer.save(directory.getCanonicalPath)
+      spark.sql(s"UPDATE delta.`${directory.getCanonicalPath}` SET value = value + 10")
+      spark.sql(s"DELETE FROM delta.`${directory.getCanonicalPath}` WHERE key = 4")
+      val inbound = Seq((3, 30)).toDF("key", "value").createOrReplaceTempView("inbound")
+      var mergeInto = s"MERGE INTO delta.`${directory.getCanonicalPath}` AS base "
```

Comment:
In Scala, we can use multiline string, such as

```
spark.sql(s"""|MERGE INTO delta.`${directory.getCanonicalPath}` AS base
                       |USING inbound
                       |ON base.key = inbound.key
                       |WHEN MATCHED THEN UPDATE SET base.value = base.value+inbound.value".stripMargin)
```
Can we add two more statements for update and delete?

```
spark.sql(s"""UPDATE delta.`${directory.getCanonicalPath}` set value = 40 where key = 1")
spark.sql(s"DELETE FROM delta.`${directory.getCanonicalPath} WHERE key = 1")
```